db/migrate/20220105173124_use_text_for_long_subscriber_lists_columns.rb

```diff
@@ -0,0 +1,15 @@
+class UseTextForLongSubscriberListsColumns < ActiveRecord::Migration[6.1]
+  def up
+    change_table :subscriber_lists, bulk: true do |t|
+      t.change :title, :text
+      t.change :url, :text
+    end
+  end
+
+  def down
+    change_table :subscriber_lists, bulk: true do |t|
+      t.change :title, :text
+      t.change :url, :text
```

Comment:
Should these `:text`s be `:string` in this `down` migration?

Reply:
Oops, yes. Thanks for catching.